### ISSUE TITLE
Any chance of compiling oh-my-posh for ARM32/ARM64?

### ISSUE THREAD

**Issue by DennisGaida** [JanDeDobbeleer/oh-my-posh#323]
### Prerequisites

- [x] I have read and understand the `CONTRIBUTING` guide
- [x] I looked for duplicate issues before submitting this one

### Description

Trying to run oh-my-posh on a Raspberry PI doesn't work: `/usr/local/bin/oh-my-posh: cannot execute binary file: Exec format error`. This is because right now there is no ARM-compiled version of oh-my-posh.

### Steps to Reproduce

1. Try to run posh-linux-amd64 on an ARM system

### Possible solution
I tried building my own version `posh-linux-arm` by setting the `GOARCH` to `arm` and everything (well I tested `oh-my-posh --print-shell` so far) seems to be working. Amend the build action to include arm/arm64?

**Comment by JanDeDobbeleer** [comment]
That's an option. I'm also adding x86 for Windows to the abilities so I can look at this at the same time.

**Comment by DennisGaida** [comment]
FYI: I'm running oh-my-posh with config and everything on a raspberrypi (ARM). It works 👍 

I found one issue with terminal multiplexer (tmux) and `alignment:right` which destroys window borders. Can be seen in the honukai theme. I'll open a separate issue for this.

**Comment by JanDeDobbeleer** [comment]
@DennisGaida  took me a few tries, but it should be there, at least the arm64 one (we can also add 32 of needed). Can you validate the binary in release [3.69.0](https://github.com/JanDeDobbeleer/oh-my-posh3/releases/tag/v3.69.0)?

**Comment by DennisGaida** [comment]
Can't test the arm64 version since my raspberrypi needs the arm version (and I tried myself with `GOARCH=arm` which works):

```
$  ./posh-linux-arm64 --print-shell
-bash: ./posh-linux-arm64: cannot execute binary file: Exec format error
```

**Comment by JanDeDobbeleer** [comment]
@DennisGaida I'll adjust towards that `GOARCH` 👍🏻

**Comment by JanDeDobbeleer** [comment]
@DennisGaida available under [v3.69.2 ](https://github.com/JanDeDobbeleer/oh-my-posh3/releases/tag/v3.69.2)

**Comment by DennisGaida** [comment]
Works!

```
$  ./posh-linux-arm --print-shell
bash
$  ./posh-linux-arm -version
3.69.2
```
